app/templates/flatpages/privacy-policy.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>Privacy Policy | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<space class="large"></space>
+<div class="content" style="margin-bottom: 90px;">
+  <h1 style="text-align: center; margin-bottom: 40px;">Privacy Policy</h1>
+  <ul>
+    <li>HackItForward stores a small amount of data such as your username, your profile picture and other user-generated data on the platform when you create an account.</li>
+    <li>Google and Github OAuth are used to facilitate the registration process.</li>
```

Comment:
These are optional and should be noted as such, as well as the consequences of signing up with them.